foundry.md

```diff
@@ -96,6 +96,29 @@ First we have some helpers in K which can:
              ...
          </account>
 ```
+```k
+
+    rule [call.deal]:
+         <k> CALL _ CHEAT_ADDR 0 ARGSTART _ARGWIDTH 0 0
+          => #setBalance(#asWord(#range(LM, ARGSTART +Int 4, 32)), #asWord(#range(LM, ARGSTART +Int 36, 32)))
+          ~>1 ~> #push
+         ...
+         </k>
+         <output> _ => .ByteArray </output>
+         <localMem> LM </localMem>
+      requires CHEAT_ADDR ==Int #address(FoundryCheat)
+       andBool #asWord(#range(LM, ARGSTART, 4)) ==Int 3364511341 // selector ( "deal" )
+      [priority(40)]
+
+    syntax KItem ::= "#setBalance" "(" Int "," Int ")" [klabel(foundry_setBalance)]
+ // -------------------------------------------------------------------------------
+    rule <k> #setBalance(ACCTID, NEWBAL) => 1 ... </k>
+         <account>
+             <acctID> ACCTID </acctID>
+             <balance> _ => NEWBAL </balance>
+             ...
+         </account>
+```
```

Comment:
Wait a second, why are we pushing `1` twice? We already do this once above, then again here. This `1` is probably just discarded. We should just do `.` on the RHS here.